bugbug/models/devdocneeded.py

```diff
@@ -73,6 +73,9 @@ def get_labels(self):
 
             for entry in bug_data['history']:
                 for change in entry['changes']:
+                    if change['field_name'] == 'keywords' and 'dev-doc-needed' in change['removed'] and 'dev-doc-complete' not in change['added']:
```

Comment:
Can you add a comment to explain the rationale? (Basically the suggestion from the issue)